code/stable/glassbr/src/cpp/designLog.txt

```diff
@@ -19,6 +19,7 @@ Successfully matched Real with Double.
 Successfully matched Boolean with Boolean.
 Successfully matched Rational with Double.
 Successfully matched Integer with Integer.
+Successfully matched Function (String :| [Real,Real]) Real with Func [String] Double.
```

Comment:
This seems like an odd match? These don't seem like comparable types.

Reply:
#3411 was a previous issue, `make stabilize` inadvertently let us find it!